### DISCUSSION TITLE
Move WRAP feature inside code viewer

### DISCUSSION THREAD

**Discussion by virtualdj** [community/community#3314] [Mobile]
Please move the **wrap code** feature with a switch or button in the action bar on the Code Viewer activity.
There are some files which are more comfortable to view with wrap mode off, while others with wrap mode on.
Going back to the settings every time is disappointing!

**Comment by candyho** [comment]
Hey, thanks for the feedback. We hear you. We have an internal issue to keep track of this request and will evaluate priority from time to time. Hopefully we'll have some updates to share soon!

**Reply by vic4key** [reply]
Hope we can get it soon. Currently, a little bit inconvenient. Thank you so much.
